common/scala/src/main/scala/whisk/core/mesos/MesosTask.scala

```diff
@@ -0,0 +1,178 @@
+      val containerIp = new ContainerAddress(taskHost, taskPort)
+      val containerId = new ContainerId(taskId);
+      new MesosTask(containerId, containerIp, ec, log, taskId, mesosClientActor, mesosConfig)
+    })
+
+  }
+
+}
+
+object JsonFormatters extends DefaultJsonProtocol {
+  implicit val createContainerJson = jsonFormat3(CreateContainer)
+}
+class MesosTask(override protected val id: ContainerId,
+                override protected val addr: ContainerAddress,
+                override protected val ec: ExecutionContext,
+                override protected val logging: Logging,
+                taskId: String,
+                mesosClientActor: ActorRef,
+                mesosConfig: MesosConfig)
+    extends Container {
+
+  implicit val e = ec
+
+  /** Stops the container from consuming CPU cycles. */
+  override def suspend()(implicit transid: TransactionId): Future[Unit] = {
+    //suspend not supported
+    Future.successful(Unit)
+  }
+
+  /** Dual of halt. */
+  override def resume()(implicit transid: TransactionId): Future[Unit] = {
+    //resume not supported
+    Future.successful(Unit)
+  }
+
+  /** Completely destroys this instance of the container. */
+  override def destroy()(implicit transid: TransactionId): Future[Unit] = {
+    mesosClientActor
+      .ask(DeleteTask(taskId))(MesosTask.taskDeleteTimeout)
+      .mapTo[TaskStatus]
+      .map(taskStatus => {
+        //verify that task ended in TASK_KILLED state (but don't fail if it didn't...)
+        if (taskStatus.getState != TaskState.TASK_KILLED) {
+          logging.error(this, s"task kill resulted in unexpected state ${taskStatus.getState}")
+        } else {
+          logging.info(this, s"task killed ended with state ${taskStatus.getState}")
+        }
+      })
+  }
+
+  /** Obtains logs up to a given threshold from the container. Optionally waits for a sentinel to appear. */
+  /** For Mesos, this log message is static per container, just indicating that mesos logs can be found via the mesos UI. */
+  /** To disable this message, and just store an empty log message per activation, set whisk.mesos.mesosLinkLogMessage=false */
+  val logMsg =
+    s"Logs are not collected from Mesos containers currently. " +
+      s"You can browse the logs for Mesos Task ID ${taskId} using the mesos UI at ${mesosConfig.masterPublicUrl
+        .getOrElse(mesosConfig.masterUrl)}"
+  val tsFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'")
+  override def logs(limit: ByteSize, waitForSentinel: Boolean)(
+    implicit transid: TransactionId): Source[ByteString, Any] =
+    if (mesosConfig.mesosLinkLogMessage) {
+      Source
+        .fromIterator(() =>
+          Iterator(ByteString.fromString(s"""{\"log\":\"${logMsg}\",\"stream\":\"stdout\",\"time\":\"${LocalDateTime
+            .now()
+            .format(tsFormat)}\"}""")))
+    } else {
+      Source.empty
```

Comment:
should perhaps generate a similar message here but without the link: sorry not supported or not configured correctly, check with your service administrator... better than no logs at all.

Reply:
I originally had this message as a configuration option (not just link or no link); I can change it to log a non-linking static message.